packages/app/ios/Podfile.lock

```diff
@@ -537,9 +586,13 @@ SPEC CHECKSUMS:
   RNScreens: f7ad633b2e0190b77b6a7aab7f914fad6f198d8d
   RNSVG: ce9d996113475209013317e48b05c21ee988d42e
   Toast: 91b396c56ee72a5790816f40d3a94dd357abc196
-  Yoga: 4bd86afe9883422a7c4028c00e34790f560923d6
+  Yoga: 8c8436d4171c87504c648ae23b1d81242bdf3bbf
   YogaKit: f782866e155069a2cca2517aafea43200b01fd5a
 
+<<<<<<< HEAD
+PODFILE CHECKSUM: 55e3aa728a80834bbc3ec778624aaa8d3cd962e0
+=======
 PODFILE CHECKSUM: 23e17da66eb2cdc077f0aa44ac22823b0e83ec75
+>>>>>>> 94dded20a679bfd56f0352eb39d443cbe6a32e64
```

Comment:
Unhandled merge conflict, which checksum is correct?

Reply:
Fixed!